doc/tutorials/using_grub.rst

```diff
@@ -0,0 +1,188 @@
+      ``yyyyyy`` is its tag and must exactly match the ``kernel_mod_tag`` of VM1 in the
+      ``hypervisor/scenarios/$(SCENARIO)/vm_configurations.c`` file.
+
+      The guest kernel command line arguments is configured in the hypervisor source code by default
+      if no ``$(VMx bootargs)`` is present. If ``$(VMx bootargs)`` is present, the default
+      command line arguments is overridden by the ``$(VMx bootargs)`` parameters.
+
+      The ``$(SOS bootargs)`` parameter in the multiboot command is appended to the end of Service
+      VM kernel command line. This is useful to override some SOS kernel cmdline parameters because
+      the later one would win if same parameters configured in Linux kernel cmdline. For example,
+      adding ``root=/dev/sda3`` will override the original root device
+      to ``/dev/sda3`` for the SOS kernel.
+
+      All parameters after ``#`` character are ignored since GRUB treat them as comments.
+
+      ``\``, ``$``, ``#`` are special characters in GRUB, a escape character ``\`` must be added
+      before these special characters if they are included in ``$(HV bootargs)`` or ``$(VM bootargs)``.
+      For example, ``memmap=0x200000$0xE00000`` for guest kernel cmdline
+      must be written as ``memmap=0x200000\$0xE00000``
+
+
+#. Update GRUB::
+
+   $ sudo update-grub
+
+#. Reboot the platform. On the platform's console, Select the
+   **Boot ACRN hypervisor xxx** entry to boot the ACRN hypervisor.
+   The GRUB loader will boot the hypervisor, and the hypervisor will
+   start the VMs automatically.
+
+
+Installing self-built GRUB
+**************************
+
+If the GRUB version on your platform is outdated or has issues booting
+the ACRN hypervisor, you can have a try with self-built GRUB binary. Get
+the latest GRUB code and follow the `GRUB Manual
+<https://www.gnu.org/software/grub/manual/grub/grub.html#Installing-GRUB-using-grub_002dinstall>`_
+to build and install your own GRUB, and then follow the steps described
+earlier in `pre-installed-grub`_.
+
+
+Here we provide another simple method to build GRUB in efi application format:
+
+#. Make GRUB efi application:
+
+   .. code-block:: none
+
+      $ git clone https://git.savannah.gnu.org/git/grub.git
+      $ cd grub
+      $ ./bootstrap
+      $ ./configure --with-platform=efi --target=x86_64
+      $ make
+      $ ./grub-mkimage -p /EFI/BOOT -d ./grub-core/ -O x86_64-efi -o grub_x86_64.efi \
+            boot efifwsetup efi_gop efinet efi_uga lsefimmap lsefi lsefisystab \
+            exfat fat multiboot2 multiboot terminal part_msdos part_gpt normal \
+            all_video aout configfile echo file fixvideo fshelp gfxterm gfxmenu \
+            gfxterm_background gfxterm_menu legacycfg video_bochs video_cirrus \
+            video_colors video_fb videoinfo video net tftp
+
+   This will build a ``grub_x86_64.efi`` binary in the current directory, copy it to ``/EFI/boot/`` folder
```

Comment:
I guess this EFI partition is what is typically mounted under `/boot`? If true, can you mention that?